### ISSUE TITLE
training loss の方が validation loss よりも高くなる原因を調査

### ISSUE THREAD

**Issue by TaiseiHashimoto** [TaiseiHashimoto/auto-trader#3]

**Comment by TaiseiHashimoto** [comment]
正例は train の方が多く、また ROC-AUC は train の方が高い。
そのためラベル分布の差から loss に差が出ていると考えられる。
